src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -565,12 +565,27 @@ export default class DefaultDeviceController
     return this.videoInputQualitySettings;
   }
 
-  acquireAudioInputStream(): Promise<MediaStream> {
-    return this.acquireInputStream('audio');
+  async acquireAudioInputStream(): Promise<MediaStream> {
```

Comment:
Nice to split them 👍

Reply:
Nice suggestion, since the device controller now has two separate "start" APIs for audio and video!

Reply:
> Destroy change is missing?

I dropped it for now as it causes the update in all docs file (not sure why yet).